lib/BenchRunner.hs

```diff
@@ -450,6 +475,8 @@ mainWith ::
     -> (String -> String -> Maybe Quickness)
     -> (String -> String -> String)
     -> IO ()
+
+-- XXX Use a defaultConfig record instead
```

Comment:
There is a `defaultConfig`. Aren't I using it (Indirectly)?

Reply:
That is internal defaultConfig. I meant using a defaultConfig for the mainWith user and override it as needed.